csun-metalab/nomi
# Hotfix croppa 6 4 18

# Summary
Clean-up the image upload code as well as the back-end component.

# To Test
The functionality will remain the same no issues should be reported on the network or console ion the Chrome dev tools.
Check image upload and refresh after to see the changes.
